zerver/openapi/zulip.yaml

```
@@ -9002,6 +9002,18 @@ paths:
                   example: true
                 content:
                   $ref: "#/components/schemas/OptionalContent"
+                prev_content_sha256:
+                  description: |
+                    The sha256 hash of the previous raw content of the
```

The name of this hash function is "SHA-256":
```suggestion
                    The SHA-256 hash of the previous raw content of the
```

See e.g. https://en.wikipedia.org/wiki/SHA-2 .

(In general, one of the things Wikipedia typically is excellent at is getting the names of things exactly pedantically correct. When in doubt about the canonical name for something, or its spelling or capitalization etc., I recommend looking it up in Wikipedia. Especially good is if the answer is the title of a Wikipedia article. In this case it's one of the bolded names in the intro of the article, because all the hash functions in this family share a single article.)